src/components/SupportTable/SupportTable.jsx

```diff
@@ -35,8 +36,7 @@ export default function SupportTable({helpRequests}) {
 							<td className="govuk-table__cell">{helpRequest.callbackRequired && <>Callback</>}</td>
 							<td className="govuk-table__cell">{helpRequest.helpRequestCalls?.length}</td>
 							<td className="govuk-table__cell govuk-table__cell--numeric">
-								{/* <a href="/helpcase-profile/1/call-starter">View</a>  */}
-								<a href="#">View</a> 
+								<a href="/helpcase-profile/1/call-starter">View</a> 
```

Comment:
Should we leave 'view' pointing to nothing until we someone starts working on that page?

Reply:
I guess so